zero-module.yml

```diff
@@ -80,3 +80,9 @@ parameters:
   - field: randomSeed
     label: Random seed that will be shared between projects to come up with deterministic resource names
     execute: uuidgen | head -c 8
+  - field: fileUploads
+    label: Enable file uploads?
```

Comment:
Maybe a bit more here? "Enable file uploads using S3 and Cloudfront signed URLs? (Will require manual creation of a Cloudfront keypair in AWS)"

Reply:
I'm all for more descriptive labels. :+1: